share/juci.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=juCi++
+Comment=A lightweight IDE
+Exec=juci %F
+Terminal=false
+Type=Application
+StartupNotify=true
+MimeType=text/plain;
```

Comment:
Is this ok:
```
Categories=Development;GTK;
```
I see Development is commonly used for applications related to programming. GTK since its main GUI library is GTK.